configure

```diff
@@ -2124,6 +2130,26 @@ EOF
 	fi
 }
 
+# Check for an executable in PATH. Takes the name of the EXE and sets
+# two variables with whether the exe exists and the path.
+# For example "test_exe_in_path curl" will set _curlpath and _curl
+test_exe_in_path() {
```

Comment:
What is _exe_ here? And why do this at all? The proper test would be `command -v pandoc >/dev/null 2>&1 && _pandoc = 'yes'` in place you're looking for pandoc.